rpcd/etc/openstack_deploy/env.d/nova.yml

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
I don't think we can filter out compute hosts in the ``ensure_local_checks.yml`` task file, since that task was built to be a generic that is used for each check defined in defaults/main.yml here.
I'm not sure if we can specificity that negate expression here: https://github.com/rcbops/rpc-openstack/blob/master/rpcd/playbooks/roles/rpc_maas/defaults/main.yml#L396. That would result in the task file conditional on line 27 to evaluate to ``- inventory_hostname in groups["neutron_l3_agent:!compute_hosts"]``. Is that possible?